pom.xml

```diff
@@ -141,6 +141,17 @@
     <dependencies>
 
         <!-- Unit tests/mocking -->
+        <dependency>
+	    <groupId>org.openjdk.jmh</groupId>
+	    <artifactId>jmh-core</artifactId>
+	    <version>1.32</version>
```

Comment:
Good. Is this the latest version?

Reply:
No - looks like it's actually 1.37: https://mavenlibs.com/maven/dependency/org.openjdk.jmh#:~:text=jmh%20%2D%20All%20Dependencies-,jmh%2Dcore%201.37,-%40org.openjdk.jmh
I'll update it now